tests/test_compationfilter.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2014 Tyler Neely
```

Comment:
Sorry for my pickyness, but now is 2019 year 😊

Reply:
No need to apologize. Thanks for catching it.  Fixed.